<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::report::worksummary::SelectUsersDialog</class>
 <widget class="QDialog" name="tt3::report::worksummary::SelectUsersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select users</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-report-worksummary.qrc">
    <normaloff>:/tt3-report-worksummary/Resources/Images/Actions/BrowseLarge.png</normaloff>:/tt3-report-worksummary/Resources/Images/Actions/BrowseLarge.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QListWidget" name="chosenUsersListWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="chosenUsersLabel">
     <property name="text">
      <string>Users to include into the report:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="allUsersListWidget"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="usersToIncludeLabel">
     <property name="text">
      <string>Available users:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tt3-report-worksummary.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::report::worksummary::SelectUsersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::report::worksummary::SelectUsersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
